packages/functional-tests/pages/login.ts

```diff
@@ -411,9 +405,9 @@ export class LoginPage extends BaseLayout {
   }
 
   async waitForPasswordHeader() {
```

Comment:
I think we also renamed an above function to just `cookiesDisabledHeader` and losing the `waitFor` prefix. Not sure if we want to do that here too. 🤷 

```diff
-async waitForCookiesDisabledHeader() {
+async cookiesDisabledHeader() {
```

Reply:
Actually we already have a PasswordHeader() hence couldn't change this.